### PR TITLE
Readme error log consent

### PR DESCRIPTION
**Pull request by braelyn-ai**
image

Mostly generated by GPT. The key points are that we adhere to CCPA requirements and that the SDK is subject to our Privacy Policy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,3 +120,9 @@ Our mission is to bring your agent from protype to production.
 Agent developers often work with little to no visibility into agent testing performance. This means their agents never leave the lab. We're changing that. 
 
 AgentOps is the easiest way to evaluate, grade, and test agents. Is there a feature you'd like to see AgentOps cover? Just raise it in the issues tab, and we'll work on adding it to the roadmap.
+
+### Disclaimer on Error Reporting
+
+_By implementing this SDK, you acknowledge and agree that the SDK may automatically send error reports to our servers. These reports are strictly limited to information relevant for identifying and resolving issues, such as error messages, stack traces (including file paths that may include your system username), and the SDK version. Our primary aim in collecting this data is to enhance the developer experience by identifying, diagnosing, and fixing issues more efficiently. No personal information or sensitive data will be transmitted as part of these reports without your explicit consent._
```
Comment (siyangqiu) [inline README.md]: This may or may not be true depending on what the stack traces contain. Maybe say that this information will only be used by us

**Review by siyangqiu — APPROVED** [review]
One comment, otherwise, good to me

File: README.md (hunk 1 above)
Comment (siyangqiu) [reply README.md]: Specifically referencing: "No personal information or sensitive data will be transmitted as part of these reports without your explicit consent."

**Review by areibman — CHANGES_REQUESTED** [review]
Let's have a `TOS.md` instead of putting this in `README.md`

**Comment by areibman** [comment]
> Let's have a `TOS.md` instead of putting this in `README.md`

I looked at some other repos, and TOS is not a common readme (sorry for the misleading suggestion). Let's keep these changes though; I've raised an issue to the  documentation repository in the meantime.

**Review by areibman — APPROVED** [review]
Thanks Braelyn :)

**Pull request merged**